src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -25,7 +27,11 @@
 
 
 <div class="container-repeatable-elements">
-    <div data-repeatable-holder="{{ $field['name'] }}"></div>
+    <div
+        data-repeatable-holder="{{ $field['name'] }}"
+        init-rows="{{$field['init_rows']}}"
+        max-rows="{{var_export($field['max_rows'])}}"
```

Comment:
I would prefix these with `data` just to be consistent and compliant.

If we also change to `min` and `max` it'll look a bit cleaner too:
```suggestion
        data-min-rows="{{ $field['min'] }}"
        data-max-rows="{{ var_export($field['max']) }}"
```

Reply:
Also, I notice there's a slight difference between the two - one is `var_exported`, one is not. Is this intentional?

Reply:
It was a leftover, I started this PR with `max_rows => false` to account for infinite number of maximum rows, then after me and @promatik discussed it, we decided to go with `0` as infinite.

Just removed it.